hat/assets/js/apps/Iaso/domains/registry/components/Instances.tsx

```diff
@@ -91,6 +94,29 @@ export const Instances: FunctionComponent<Props> = ({
     const currentForm: Form | undefined = useMemo(() => {
         return formsList?.find(f => `${f.value}` === formIds)?.original;
     }, [formIds, formsList]);
+
+    useEffect(() => {
+        if (
+            formsList?.length &&
+            formsList?.length >= 0 &&
```

Comment:
For same effect, but shorter you can do 
```suggestion
            (formsList?.length ?? 0)>= 0 &&
```

We have the polyfill for `??` now anyway 😁